assignment.js

```diff
@@ -57,13 +87,24 @@ Use the following test cases to confirm your program meets the success criteria
 2. Given the following:
     problem1Numbers = [9, 18, 62, 3, 17]
 
-   Result: 0
+   Result: 21.8
 
 ************/
 
 console.log('Problem 2:')
 
 // Add your code below this line
+let values = [1, 2, 3]
+let sum = values.reduce((previous, current) => current += previous);
```

Comment:
Great way to use reduce method, but be sure to set an initial value when using this method. Ex. let sum = values.reduce((acc, curr) => acc += curr, 0)